### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Rupamdas832**
Auth (Login/Signup) is from backend(monogo-db and mongoose)
User persist locally
Responsive too.
Functionality for storing video data locally is pending

### CODE REVIEW

File: src/Pages/History/History.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+export const History = () => {
+
+    const {storeState} = useStore()
+    const {historyVideos} = storeState
+    return (
+        <div className="historyContainer">
+        <h1>History</h1>
+        <div className="historyVideoList">
+        {historyVideos.map((video) => {
```
Comment (ritikarp) [inline src/Pages/History/History.js]: In history, you can map the array in reverse order so that the last watched video comes to the top.

**Review by ritikarp — COMMENTED** [review]
Hi Rupam,  kudos on the work done.
Added a comment please check. The folder name Store could be renamed to make is more understandable.
Also, add responsiveness for devices like iPads. 
You can set cards to stretch for equal height.
Rest LGTM

File: src/Pages/Signup/Signup.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.signupContainer{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```
Comment (neharajesh) [inline src/Pages/Signup/Signup.css]: You can avoid writing these three lines by just writing "place-items: center"!

File: src/Pages/VideoDetail/VideoDetail.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
+.videoDetailContainer{
+    padding: 2rem 0 0 1rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
```
Comment (neharajesh) [inline src/Pages/VideoDetail/VideoDetail.css]: "place-items: center" works here too!

**Review by neharajesh — COMMENTED** [review]
Hey Rupam! 
Video Library is looking good! 
I've suggested a change which would help you write lesser lines of code. Hope that helps!
Wrt to the UI - adding ellipses to the video title if it exceeds 2 lines would be better IMO and a small margin on the left of the account image would make it look better! 
Everything else is amazing, good job!

File: src/Pages/Signup/Signup.css (hunk 2 above)
Comment (Rupamdas832) [reply src/Pages/Signup/Signup.css]: Thanks Neha. But It didn't work with just one line. I still have to write "justify-content: center" to get the desired output

**Pull request merged**